go.sum

```diff
@@ -40,6 +40,8 @@ github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/dnaeon/go-vcr v1.0.1 h1:r8L/HqC0Hje5AXMu1ooW8oyQyOFv4GxqpL0nRP7SLLY=
 github.com/dnaeon/go-vcr v1.0.1/go.mod h1:aBB1+wY4s93YsC3HHjMBMrwTj2R9FHDzUr9KyGc8n1E=
+github.com/fastly/go-fastly v1.17.0 h1:N92pKRkGVWt57f3zMeWijjHnYvFOtT1QvYXhE+NFIhA=
+github.com/fastly/go-fastly v1.17.0/go.mod h1:fwYSSnZ6zEClwRS65T0f57Yh83Tc4gL12GgttQwJZfA=
```

Comment:
This doesn't seem right, why do we now reference both v1 and v2 of go-fastly?

Reply:
Interesting, maybe I didn't run the `go mod tidy` to get remove this.

Reply:
This is resolved now (`go.mod` is pointing to `v2.0.0-alpha.2` of go-fastly). Vendor and `go.sum` were updated accordingly